examples/posenet/main.js

```diff
@@ -0,0 +1,81 @@
+const util = new Utils();
+const canvasSingle = document.getElementById('canvas');
+const ctxSingle = canvasSingle.getContext('2d');
+const canvasMulti = document.getElementById('canvas_2');
+const ctxMulti = canvasMulti.getContext('2d');
+const scaleCanvas = document.getElementById('scaleImage');
+const scaleCtx = scaleCanvas.getContext('2d');
+const inputSize = [1, 513, 513, 3];
+async function drawSingleandMulti(){
```

Comment:
Please use a `changeBackend` function to handle the backend switching. See https://github.com/intel/webml-polyfill/blob/master/examples/mobilenet/main.js#L21